src/js/ui/shared/form.ts

```diff
@@ -14,15 +14,17 @@ function renderOption(label: string, value: string, storedValue: string, labelAr
 }
 
 
-function renderOptionGroup(label, value, storedValue, labelArg, labelArg2) {
+function renderOptionGroup(label:string, value:any, storedValue:string, labelArg:string, labelArg2:string): any {
```

Comment:
Why `any` for value? it should be `string`, or maybe `string | number`, but not `any`